pkg/controller/operators/olm/apiservices.go

```diff
@@ -251,8 +255,8 @@ func (a *Operator) isAPIServiceAvailable(apiService *apiregistrationv1.APIServic
 	return false
 }
 
-func (a *Operator) areAPIServicesAvailable(descs []v1alpha1.APIServiceDescription) (bool, error) {
-	for _, desc := range descs {
+func (a *Operator) areAPIServicesAvailable(csv *v1alpha1.ClusterServiceVersion) (bool, error) {
+	for _, desc := range csv.Spec.APIServiceDefinitions.Owned {
 		apiServiceName := fmt.Sprintf("%s.%s", desc.Version, desc.Group)
```

Comment:
nit: ideally would use desc.GetName() here too

Reply:
Agreed. There are a bunch of places that use it. I didn't change older stuff because I wanted to get this in quickly. I think we can phase it in with future PRs. I also wonder if we can just enforce the relation to group and version in our OpenAPI schema validation.